apps/lilt/src/emc-dialog.js

```diff
@@ -0,0 +1,337 @@
+/* global ManagementClient */
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useAsync } from 'react-use';
+import { Stack, IconButton, Heading } from '@contentful/f36-components';
+import { CloseTrimmedIcon } from '@contentful/f36-icons';
+import CMClient from './contentful-management-client';
+
+const BASE_URL = process.env.CONNECTORS_BASE_URL;
```

Comment:
These environment variables (`CONNECTORS_BASE_URL`, `HYDRA_CLIENT_ID`, and `HYDRA_CLIENT_STATE`) were likely configured in your repo where this app was hosted previously, but are not configured here.

Reply:
@whitelisab Yes, let me get back to you with those values. We were using Kubernetes to handle the env vars.